### DISCUSSION TITLE
Daily Standup - Thursday, May 4

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#166] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by Int-Angel** [comment]
**What did you achieve in the last 24 hours?**:
 - PR board page
 - Card chart
 - Chart menu

**What are your priorities for the next 24 hours?**:
 - Implement the 3 charts from tremor

**Blockers**:
 -

**Shoutouts**:
 - @DavBot02 being awesome

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Started working on the presentation for today
 - Read some tRPC documentation
 - Spotted a problem with our environments

**What are your priorities for the next 24 hours?**:
 - Work in the auth feature with @DavBot02 
 - Create staging environment in vercel
 - Create a local development workflow

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on User repository
 - Worked on passport auth
 - Worked on presentation

**What are your priorities for the next 24 hours?**:
 - Work on auth

**Blockers**:
 - Not sure hhow to implement some functions

**Shoutouts**:
 - @Int-Angel for presenting some interesting hypothetical scenarios
